News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/NewsReaderListActivity.java

```diff
@@ -1253,6 +1252,37 @@ private void openRssItemInExternalBrowser(Uri currentUrl) {
 		startActivity(browserIntent);
 	}
 
+	private void prepareAccountMenuItem(MenuItem accountMenuItem) {
+		if (currentUser == null || currentUser.getId() == null) {
+			// the default menu item is fine if no user info is present
+			return;
+		}
+
+		accountMenuItem.setTitle(currentUser.getDisplayName());
```

Comment:
This line replaces the tooltip line from before. Long press on the item reveals the user name.